src/handler.ts

```diff
@@ -28,12 +28,11 @@ const handler = async (event: ScheduledEvent): Promise<{ statusCode: number; bod
     await sendEvents(facilitySchedules);
 
     logger.info('Data processed successfully.');
-    return { statusCode: 200, body: 'Data processed successfully.' };
+    callback(null, 'Data processed successfully.');
```

Comment:
does the callback provide a 200 status when successful?

Reply:
It doesn't. I'm not sure how eventbridge calls a lambda, but the statusCode is not needed.

Reply:
Will this mess with manual invocations with regards to response codes ?

Reply:
No. This change has no change to the response codes being returned.